gcloud/storage/bucket.py

```diff
@@ -103,7 +103,7 @@ def client(self):
         """The client bound to this bucket."""
         return self._client
 
-    def blob(self, blob_name, chunk_size=None):
+    def blob(self, blob_name, chunk_size=None, generation=None):
```

Comment:
Reasoning: `generation` is a property of a blob, but there are too many properties to provide in the constructor